jcoin.datacommons.io/portal/gitops.json

```diff
@@ -146,6 +146,130 @@
     "explorer": true
   },
   "explorerConfig": [
+    {
+      "tabTitle": "Quarterly Report",
```

Comment:
The explorer page will be broken between the time this configuration gets deployed (soon after PR merge) and the next time the ETL runs, because `project_name` will not exist. Additionally, if the ETL fails, the explorer page will stay broken. We usually recommend splitting the ETL mapping update and the portal config update into 2 PRs. Otherwise lgtm so I will approve and the team can decide how to proceed

Reply:
Thanks Pauline, I understand -- next time I will split up -- for now, can we merge (given etlmapping/gitops was fine in QA instance)?